platform-api/src/routers/content.py

```
@@ -0,0 +1,124 @@
+        {"site": site_filter, "searchTerm": search_term, "types": request.types}
+    )
+    
+    # Map 'name' to 'title' if it's a sponsor, format to SearchResult
+    results = []
+    for item in raw_results:
+        results.append({
+            "_id": item["_id"],
+            "_type": item["_type"],
+            "title": item.get("title") or item.get("name") or "Unknown",
+            "_score": item.get("_score")
+        })
```

Ruff (0.15.6)

[warning] Use a list comprehension to create a transformed list

(PERF401)